### ISSUE TITLE
[Suggestion] Option to disable Manure function like bonemeal

### ISSUE THREAD

**Issue by MasterEnderman** [GenDeathrow/Hatchery#178]
It would be really nice, if you could add a config option to prevent chicken manure functioning like bonemeal, when using it on grass, crops or saplings.

**Comment by Syndaryl** [comment]
What would be the intent of chicken manure then? Only as a crafting ingredient for manure blocks and fertilizer?

**Comment by MasterEnderman** [comment]
exactly. also this is mainly for modpack makers if they want to get rid of all "bonemeal" like items

**Comment by GenDeathrow** [comment]
yeah tbh Im really thinking this should have been a core function. manure was added in before liquid fertilizer. Ill have to see how I want to progress on it. I may remove it with an option to turn it back on instead.
